src/sentry/rules/conditions/new_high_priority_issue.py

```diff
@@ -34,26 +33,19 @@ def passes(self, event: GroupEvent, state: EventState) -> bool:
     def get_activity(
         self, start: datetime, end: datetime, limit: int
     ) -> Sequence[ConditionActivity]:
-        # reappearances are recorded as SET_UNRESOLVED with no user
-        activities = (
-            Activity.objects.filter(
+        first_seen = (
+            Group.objects.filter(
                 project=self.project,
-                datetime__gte=start,
-                datetime__lt=end,
-                type__in=[ActivityType.SET_UNRESOLVED.value],
-                user_id=None,
+                first_seen__gte=start,
+                first_seen__lt=end,
+                priority=PriorityLevel.HIGH,
             )
-            .order_by("-datetime")[:limit]
-            .values_list("group", "datetime", "data")
+            .order_by("-first_seen")[:limit]
```

Comment:
why are we changing from `-datetime` to `first_seen`?

Reply:
We're querying the Group table to get new groups with priority - new groups don't have an activity set for this. The Group table does not have a `datetime` column.